### PR TITLE
add 2 examples and update readme

### PR DESCRIPTION
**Pull request by kyliel**
## Purpose
1. add 2 examples
2. update readme
* ...

## Does this introduce a breaking change?
No

## Pull Request Type

[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
ansible-playbook xx.yml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,30 @@
-# Project Name
+# Ansible-playbooks
```
Comment (zikalino) [inline README.md]: I think we could just have "Ansible Playbooks for Azure" in title here

File: VMCreate.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+- name: Create Azure VM
+  hosts: localhost
+  connection: local
+  vars:
+    resource_group: demo
```
Comment (zikalino) [inline VMCreate.yml]: I think we could add comment here:
# change variables below to customize your VM deployment
or something like this

Comment (zikalino) [reply VMCreate.yml]: haha, i have added # at the beginning of the line, that's why so big ;-)

File: VMCreate.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
+---
```
Comment (zikalino) [inline VMCreate.yml]: perhaps we should have comment here to describe what the playbook does in more details

File: VMCreate.yml (hunk 2 above)
Comment (yaweiw) [reply VMCreate.yml]: love the size of font!

Comment (kyliel) [reply VMCreate.yml]: Thank you for suggestion. Add comment for 2 playbook files.

File: VMCreate.yml (hunk 3 above)
Comment (kyliel) [reply VMCreate.yml]: update

**Comment by kyliel** [comment]
@zikalino  ready_for_review

**Comment by kyliel** [comment]
#sign-off

**Pull request merged**